src/test/java/com/google/api/codegen/testdata/discoveries/ruby/ruby_taskqueue.v1beta2.json.baseline

```diff
@@ -5,31 +5,27 @@
 #    and check the quota for your project at
 #    https://console.developers.google.com/apis/api/taskqueue
 # 2. Install the Ruby client library by running `gem install google-api-client`
+
 require 'googleauth'
 require 'google/apis/taskqueue_v1beta2'
 require 'json'
 
 service = Google::Apis::TaskqueueV1beta2::TaskqueueService.new
 
-# TODO: Change placeholder below to get authentication credentials.
-# See: https://foo.com/bar
+# TODO: Change placeholder below to generate authentication credentials. See:
+# https://foo.com/bar
 #
-# To use the `get_taskqueue` method, authorize using one of the following scopes:
-#   https://www.googleapis.com/auth/taskqueue
-#   https://www.googleapis.com/auth/taskqueue.consumer
+# Authorize using one of the following scopes:
+#    https://www.googleapis.com/auth/taskqueue
```

Comment:
I think the relative indent should be the same as the indent tab width for the language. Right now Java and Node.js have a relative indent of 3 spaces, like that above, but I think all these should be 2. (C# and Python should have 4.)